### PR TITLE
Add Discord Canary

### PR DESCRIPTION
**Pull request by QORTEC**
**Discord Canary**
Site: https://discordapp.com
Linux Alpha: https://www.reddit.com/r/discordapp/4bu7lm

**Solus Bug Tracker**
Bug 844: https://bugs.solus-project.com/show_bug.cgi?id=844

### CODE REVIEW

File: network/im/discord-canary/pspec.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            <Dependency>gconf</Dependency>
+        </RuntimeDependencies>
+        <Name>discord-canary</Name>
+        <Icon>discord-canary</Icon>
+        <Files>
+            <Path fileType="executable">/usr/bin</Path>
+            <Path fileType="data">/usr/share/applications</Path>
+            <Path fileType="data">/usr/share/discord-canary</Path>
+            <Path fileType="doc">/usr/share/doc</Path>
+            <Path fileType="data">/usr/share/lintian</Path>
```
Comment (ikeydoherty) [inline network/im/discord-canary/pspec.xml]: Not a valid Solus path

**Comment by ikeydoherty** [comment]
This should be a pull request with a single, tested, working commit, to add the functionality to the repo, not multiple commits.

Before we even consider a rebase here, please confirm if Discord is an electron app or not.

**Comment by feskyde** [comment]
Yes, it's an electron app

**Comment by ikeydoherty** [comment]
Then I see no point in merging it.

**Comment by QORTEC** [comment]
Yes I believe that its is an electron app.

**Comment by ikeydoherty** [comment]
Does it actually add anything that the website doesn't feature, outside of the standard tray icon and window that doesnt close?

**Comment by QORTEC** [comment]
Push to Talk:
screenshot from 2016-07-18 16-07-29

**Comment by ikeydoherty** [comment]
That's a fairly compelling reason. lol

**Comment by QORTEC** [comment]
I "cleaned" up the commit history.

File: network/im/discord-canary/pspec.xml (hunk 1 above)
Comment (QORTEC) [inline network/im/discord-canary/pspec.xml]: If there is a issue is with line 30 ``/usr/share/lintian`` then I do not know how to resolve the issue.

When I was looking how to handle **lintian** I came across **n1** project in the git.solus-project.com repository, and they also use the same path for  **lintian**  I use.  - [n1 pspec_x86_64.xml](https://git.solus-project.com/packages/n1/tree/pspec_x86_64.xml)

I asked @JoshStrobl about the **lintian** path on IRC and to my understanding of what he said; it is a valid platform independent path.

I don't know how to proceed, any information would be helpful.

**Comment by ikeydoherty** [comment]
LGTM, thanks!

**Pull request merged**